obspy/core/util/testing.py

```diff
@@ -192,7 +283,7 @@ def __init__(self, image_path, image_name, reltol=1, *args, **kwargs):
         self.keep_only_failed = "OBSPY_KEEP_ONLY_FAILED_IMAGES" in os.environ
         self.output_path = os.path.join(image_path, "testrun")
         self.diff_filename = "-failed-diff.".join(self.name.rsplit(".", 1))
-        self.tol = get_matplotlib_defaul_tolerance() * reltol
+        self.tol = reltol * 3.0
```

Comment:
Ah, I see this "fixes" the station response plots, but those are legitimately broken (just not in ObsPy.)